### PR TITLE
Raghav/homepage db

### PR DESCRIPTION
**Pull request by raghavthakur**
- Rewrote search bar to be dynamic with real symbol lookups
- Fixed issue of not being able to run server, had to remove database connection code due to issues with mongo db
Screen Shot 2021-07-17 at 3 57 21 AM

Screen Shot 2021-07-17 at 3 59 21 AM

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+    },
+    "dependencies": {
+        "body-parser": "^1.19.0",
+        "cookie-parser": "~1.4.4",
+        "cors": "^2.8.5",
+        "debug": "~2.6.9",
+        "dotenv": "^10.0.0",
+        "express": "~4.16.1",
+        "http-errors": "~1.6.3",
+        "jade": "^1.9.2",
```
Comment (HGoyal09) [inline server/package.json]: Please do not change/downgrade these versions

File: src/components/StockSearchBar/ShowStock/index.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { useColorModeValue } from '@chakra-ui/react';
+import React from 'react';
+import styled from 'styled-components';
+
+const StockContainer = styled.div`
```
Comment (HGoyal09) [inline src/components/StockSearchBar/ShowStock/index.jsx:4]: i think we are not use html with css 
Can we please use HTML with css for consistency 
+ we are not using styled components in the project either

File: src/components/StockSearchBar/ShowStock/index.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    font-size: 15px;
+    color: textColor;
+    margin-left: 10px;
+    flex: 2;
+    display: flex;
+`;
+
+const ShowStock = (props) => {
+    const { symbol, name } = props;
+    const textColor = useColorModeValue('#000', '#fff');
```
Comment (HGoyal09) [inline src/components/StockSearchBar/ShowStock/index.jsx]: TextColor is not being used

**Review by HGoyal09 — DISMISSED** [review]

**Review by HGoyal09 — APPROVED** [review]

File: src/components/StockSearchBar/ShowStock/index.jsx (hunk 2 above)
Comment (raghavthakur) [reply src/components/StockSearchBar/ShowStock/index.jsx:4]: Sure, will refactor.

**Pull request merged**